xregexp/build.boot

```diff
@@ -0,0 +1,36 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.6.0"  :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "3.1.1")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom  {:project     'cljsjs/xregexp
+       :version     +version+
+       :description "Extended JavaScript regular expressions"
+       :url         "https://github.com/slevithan/xregexp"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+
+(deftask package []
+  (comp
+   (download :url "https://raw.githubusercontent.com/slevithan/xregexp/37413619ea9f03638bfa92e038e4b4c15645d446/src/xregexp.js"
```

Comment:
Please use `(format "https://raw.githubusercontent.com/slevithan/xregexp/v%s/src/xregexp.js" +lib-version+)` to build the url here. This way url doesn't need to be changed when version is updated.